core/classes/BugFilterQuery.class.php

```diff
@@ -1518,11 +1518,13 @@ protected function helper_sort_column_alias( $p_prop ) {
 			case 'category_id':
 				# This join will be reduced as unique, if category search is active
 				$this->add_join( 'LEFT JOIN {category} ON {bug}.category_id = {category}.id' );
+				$this->add_select( '{category}.name' );
 				return '{category}.name';
 				break;
```

Comment:
Why is break needed after return?  Applies to both cases included in the diff.

Reply:
The `break` is indeed dead code (also at line 1553)

Reply:
> Why is break needed after return?

I usually put always a `break` unless it's explicitly intended to fall through next case.
What is our guideline here?